src/pages/ResearcherReview.js

```diff
@@ -47,7 +46,7 @@ class ResearcherReview extends Component {
 
   async findResearcherInfo() {
 
-    const user = await User.getById(this.props.match.params.dacUserId);
+    const user = await User.getById(this.props.userId);
```

Comment:
I think this change breaks the "Review Researcher" button on the "Manage Users" page. I think that button is no longer necessary. Can we remove it for this PR? That would solve the problem that it doesn't work 😄

Reply:
I removed the Researcher Review column. There's currently some extra space on the right from where it used to be, but I'm not sure if I should look into that or leave it for now.

Reply:
You can leave that in place, not necessary to shrink it down.